internal/pkg/funccalls/analyzer_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2019 Google LLC
```

Comment:
Revert this line?

Reply:
Hmm. GitHub messed up, this has nothing to do with my changes.